scripts/import-cp-contexts.sh

```diff
@@ -0,0 +1,247 @@
+#!/usr/bin/env bash
+# Copyright 2024 The KubeStellar Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+arg_names=""
+arg_kubeconfig=""
+arg_context=""
+arg_ip_addr=""
+arg_merge=false
+arg_output=""
+arg_verbose=true
+
+
+display_help() {
+  cat << EOF
+Usage: $0 [--kubeconfig <filename>] [--context <name>] [--names <list of names>] [--replace-local-ip-address <address>] [--merge] [-o <filename>] [-V] [-X]
+--kubeconfig <filename>                 use the specified kubeconfig
+--context <name>                        use the specified context
+--names <name1>,<name2>                 comma separated list of KubeFlex Control Planes names to import, instead of default *all*
+--replace-local-ip-address <address>    replaces server addresses "127.0.0.1" with <address>
+--merge                                 merge the control planes contexts with the existing cluster contexts
+-o <filename>                           specify a different kubeconfig file to save the contexts (- for stdout)
+--silent                                no information output
+-X                                      enable verbose execution of the script for debugging
+EOF
+}
+
+
+get_kubeconfig() {
+    context="$1"
+    cp_name="$2"
+    cp_type="$3"
+    ip_addr="$4"
+
+    echov "Getting the kubeconfig of Control Plane \"$cp_name\" of type \"$cp_type\" in context \"$context\":"
```

Comment:
I think this would be clearer if it said "from" rather than "in".
Not a blocker.

Reply:
ok